### PR TITLE
[script.module.kutils] 1.3.0

### PR DESCRIPTION
**Pull request by fbacher**
### Description
This is my second attempt, after making, I believe, all of the recommended changes from the first attempt. This time I figured out how to do the subtree merge so that changes are tracked to my repository. Unfortunately, this destroyed the tracking of the first pull request (#2087), which I closed.

As stated in PR 2087, I am accepting the responsibility to maintain this addon (originally script.module.kodi65). It has been orphaned for some time. Among other things I believe that I have resolved a long standing issue that caused Kodi to hard crash (Limitation of numpy and embedded python).

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by fbacher** [comment]
Added fix for a regression. Had to try several times to get down to one commit

**Comment by fbacher** [comment]
Fixed file permissions identified by checker.

**Review by enen92 — CHANGES_REQUESTED** [review]
Minor. After that it's good to go

File: script.module.kutils/.gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Byte-compiled / optimized / DLL files
```
[the remark this answers is not in the document]
Comment (fbacher) [reply script.module.kutils/.gitignore:1]: Ok. I'll commit change after resolving version change issue (see comment below)

File: script.module.kutils/lib/kutils/player.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        listitem = xbmcgui.ListItem(label=vid.title)
+        listitem.setArt({'thumb': vid.thumbnail})
+        listitem.setInfo(type='video',
+                         infoLabels={"genre": vid.sourceName,
+                                     "plot": vid.description})
+        return vid.streamURL(), listitem
+
+    def wait_for_video_end(self):
+        monitor: xbmc.Monitor = xbmc.Monitor()
+        while not monitor.waitForAbort(0.1):
```
[the remark this answers is not in the document]
Comment (fbacher) [reply script.module.kutils/lib/kutils/player.py:42]: Ok. I'll commit change after resolving version change issue (see comment below)

**Comment by fbacher** [comment]
Would it cause grief if I change version to 1.3.0 at this late stage? It seems that changing the addon name as well as lib path, etc. would warrant more than a +matrix.7 suffix to the previous version. I should have thought about this earlier. 

I'm not sure what impact this would have on the PR, since it has the version embedded. I'm fine to leave the version as is to avoid complications.

**Comment by enen92** [comment]
That's fine. You don't need the matrix version identifier as this add-on will be only on the matrix branch. We use those identifiers to distinguish exact same versions in Leia and matrix.

**Comment by fbacher** [comment]
Passed very basic tests

**Review by enen92 — APPROVED** [review]

**Pull request merged**